src/Components/dokiHome/DokiHome.js

```diff
@@ -2,11 +2,9 @@ import React, { useState } from 'react';
 import { View } from 'react-native';
 import DokiEggView from './DokiEggView';
 import DokiView from './DokiView';
-import { useUserData } from '../../hooks/useUserData';
 
 const DokiHome = ({ navigation }) => {
   const { isLoading, isError, data, error } = useUserData();
```

Comment:
will throw `[ReferenceError: Can't find variable: useUserData]` because useUserData is no longer imported